### DISCUSSION TITLE
mysql全量同步，为什么内存消耗达到了30G

### DISCUSSION THREAD

**Discussion by ChPi** [apache/flink-cdc#3059] [Q&A]
并行度设置的1
scan.incremental.snapshot.chunk.size: 80960
flink cdc 3.0

分块同步的，看日志数据量一次也就几万条，怎么快照阶段内存消耗这么多

**Comment by Kryzeno** [comment]
有bug  全量数据量大的情况下 他分片取 范围会出现null的情况  导致一次查出整个库的数据 最终内存溢出

**Reply by Kryzeno** [reply]
这种bug竟然到3.0都没解决 哎 bug太多  最近升级到3.3.0 发现全量数据没反应了 直接卡了 也不报错 真的大大服了
